Cargo.toml

```diff
@@ -31,6 +31,7 @@ bitvec = { version = "1.0", default-features = false }
 config = "0.11.0"
 clap = { version = "4.5.4", features = ["derive", "env"] }
 futures = "0.3.24"
+hashbrown = "0.14.5"
```

Comment:
This is because the `wsts::state_machine::PublicKeys` struct uses the `hashbrown` hashmap, so we need this to construct that struct. A longer term solution would be to implement a constructor in `wsts` that doesn't leak the internal `HashMap` variant used, but for now we're stuck with this.